docs/source/rules/REQ-E004.rst

```diff
@@ -0,0 +1,39 @@
+[REQ-E004] - Changing additionalProperties to False for a request parameter
+=====================================================
+
+Rationale
+---------
+If the object is defined with additionalProperties set to False then the object will not allow presence of properties not defined on the properties section of the object definition. Changing additionalProperties from True to False makes objects sent from a client, that contain additional properties and were permitted by the "old" Swagger specs, to the server be considered invalid by the backend.
+
+Mitigation
+----------
+A possible mitigation could be to not set additionalProperties to False in the object schema.
```

Comment:
I would rather be more explicit here mentioning the fact that there there is no mitigation for this, but rather a recommended approach going forward.

You can find an example of it on https://swagger-spec-compatibility.readthedocs.io/en/latest/rules/MIS-E002.html#mitigation

Reply:
That makes sense - I'll change the phrasing to be more explicit.